src/citrine/resources/experiment_datasource.py

```diff
@@ -106,7 +107,8 @@ def build(self, data: dict) -> ExperimentDataSource:
 
     def list(self, *,
              per_page: int = 100,
-             branch_id: Optional[UUID] = None,
+             branch_id: Optional[Union[UUID, str]] = None,
+             branch_version_id: Optional[Union[UUID, str]] = None,
```

Comment:
The naming of this parameter was misleading, so much so that the documentation was wrong, although its use was correct.